rmw_zenoh_cpp/src/detail/liveliness_utils.hpp

```diff
@@ -270,6 +269,29 @@ struct equal_to<rmw_zenoh_cpp::liveliness::ConstEntityPtr>
     return lhs->keyexpr_hash() == rhs->keyexpr_hash();
   }
 };
+
+template<>
+struct hash<rmw_zenoh_cpp::Gid>
+{
+  std::size_t operator()(const rmw_zenoh_cpp::Gid & gid) const noexcept
+  {
+    // This function implemented FNV-1a 64-bit as the GID is small enough
+    // (e.g. as of commit db4d05e of ros2/rmw RMW_GID_STORAGE_SIZE is set to 16)
+    // and FNV is known to be efficient in such cases.
+    //
+    // See https://github.com/ros2/rmw/blob/db4d05e/rmw/include/rmw/types.h#L44
+    // and https://datatracker.ietf.org/doc/html/draft-eastlake-fnv-17.html
+    static constexpr uint64_t FNV_OFFSET_BASIS_64 = 0x00000100000001b3;
```

Comment:
Include `<cstdint>` for `uint64_t`

Reply:
Already there: https://github.com/ZettaScaleLabs/rmw_zenoh/blob/a372ede0834c69b0b649cf8f798b43a9f1c1b9c5/rmw_zenoh_cpp/src/detail/liveliness_utils.hpp#L20.